tardis/plasma/properties/atomic.py

```diff
@@ -67,8 +67,10 @@ class Lines(BaseAtomicDataProperty):
     """
     Attributes
     ----------
-    lines : Pandas DataFrame (wavelength, atomic_number, ion_number, f_ul, f_lu, level_number_lower,
-                              level_number_upper, nu, B_lu, B_ul, A_ul, wavelength)
+    lines : Pandas DataFrame 
+        Table of
+        (wavelength, atomic_number, ion_number, f_ul, f_lu, level_number_lower,
+        level_number_upper, nu, B_lu, B_ul, A_ul, wavelength)
         All atomic lines data. Index = line_id.
```

Comment:
```suggestion
        Atomic lines data. Columns are wavelength, atomic_number,ion_number,
        f_ul, f_lu, level_number_lower, level_number_upper, nu, B_lu, B_ul, A_ul,
        wavelength. Index is line_id.
```

No need to mention table - type dataframe shows that already.

Reply:
I'm not sure why they were not resolved?

Reply:
It was mentioned as the original description had the contents of each index in the DataFrame, so the description of describing what's inside the table was kept. With your comment, I thought it meant removing the "Table of"